auth/server/src/main/java/com/seagates3/controller/TempAuthCredentialsController.java

```diff
@@ -84,6 +84,11 @@
           if (response == null) {
             authenticateWithLdap(account, user);
             LOGGER.debug("LDAP Authentication successfull");
+            AccessKeyDAO accessKeyDAO =
+                (AccessKeyDAO)DAODispatcher.getResourceDAO(
+                    DAOResource.ACCESS_KEY);
+            // Perform cleanup before creating new temp accesskey
```

Comment:
@shalakadharap @bkirunge7 Can we log one liner saying "Deleting access key for < account-name >"

Reply:
I will log @sachitanands 
Confirmed with @bkirunge7